src/languages.js

```diff
@@ -22,7 +22,7 @@ i18next.init({
 				instructions: `<span class="title">Create WhatsApp Maps with Kapta </span><hr> 1 - Share locations in a WhatsApp group<br> 2 - Export chat to Kapta<br> 3 - Share your WhatsApp Map`,
 				watchtutorial: "Watch tutorial",
 				tutorialUrl:
-					"https://www.youtube.com/embed/PDOVl8smi2Q?si=CiIkw6_rm1TzmZgF",
```

Comment:
The old tutorialURL is still present in the other languages. I'd suggest just removing that entry for languages which don't have a translated video. In that case it'd fall back to english.

Reply:
yeah, but better to have an english one than none. I'm awaiting the translations for am, pt, yo. I should get them soon

Reply:
oh okay, I thought I updated the url. So now is the same url as English @tcouch